src/main/java/com/gd/clinic/entities/Event.java

```diff
@@ -0,0 +1,39 @@
+package com.gd.clinic.entities;
+
+import lombok.*;
+import org.hibernate.annotations.Type;
+
+import javax.persistence.*;
+import java.time.OffsetDateTime;
+import java.util.UUID;
+
+@NoArgsConstructor
+@RequiredArgsConstructor
+@Getter
+@Setter
+@ToString
+@Entity
+@Table(name = "Events")
+public class Event {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Type(type = "uuid-char")
+    private UUID id;
+
+    @NonNull
+    @Type(type = "uuid-char")
+    private UUID prescriptionId;
+
+    @NonNull
+    @Type(type = "uuid-char")
+    private UUID patientId;
+
+    @NonNull
+    private OffsetDateTime dateTime;
+
+    @NonNull
+    private String status;
```

Comment:
this should be enum

Reply:
correct, but backend should be as secure as possible on any levels. 
right now the only two of you are writing the application and you know every line of the code, but imagine that somebody another will join in future without any knowledge and will use it in incorrect way. it might lead to unexpected behaviour, so please make it here enum as well